frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart

```diff
@@ -62,38 +63,37 @@ class CalendarDayCard extends StatelessWidget {
           );
         }).toList();
 
-        final child = Padding(
-            padding: const EdgeInsets.all(8.0),
-            child: Column(
-              mainAxisSize: MainAxisSize.min,
-              children: [
-                _Header(
-                  date: date,
-                  isInMonth: isInMonth,
-                  isToday: isToday,
-                  onCreate: () => onCreateEvent(date),
-                ),
-                VSpace(GridSize.typeOptionSeparatorHeight),
-                Flexible(
-                  child: ListView.separated(
-                    itemBuilder: (BuildContext context, int index) {
-                      return children[index];
-                    },
-                    itemCount: children.length,
-                    separatorBuilder: (BuildContext context, int index) =>
-                        VSpace(GridSize.typeOptionSeparatorHeight),
-                  ),
-                ),
-              ],
-            ));
+        final child = Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            _Header(
+              date: date,
+              isInMonth: isInMonth,
+              isToday: isToday,
+              onCreate: () => onCreateEvent(date),
+            ).padding(top: 8.0, horizontal: 8.0),
```

Comment:
Why not wrap this in a padding widget?

Reply:
We don't have anything against this in the style guide. After all, it wasn't your PR that introduced `styled_widget`. 

Flutter uses [declarative paradigms](https://docs.flutter.dev/development/data-and-backend/state-mgmt/declarative), but the style widget package uses imperative paradigms and extensions for the sake of decreasing indentation. 
> https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension

Reply:
I made my [original review request](https://github.com/AppFlowy-IO/AppFlowy/pull/2060#discussion_r1143187860) because I haven't seen or used the package before, it looks quite odd to me. Regardless, I'd be fine landing this as is.

Reply:
The TL;DR is that wrapping this widget in another `Padding` widget is declarative, whereas this is imperative. Flutter implements declarative paradigms, and `styled_widget` violates that. But at this point, the package is in the repo, so the dog is already let off the leash.

Reply:
oh so they don't `strictly` stick to their style

Reply:
I see, I didn't know about this so thanks so much for the info. If this is deemed as bad practice, I think we should try to avoid it. I will make the changes in this PR.